### ISSUE TITLE
Creating a unique name for a SDO key pair 

### ISSUE THREAD

**Issue by amaya-Mali** [open-horizon/SDO-support#101]
@bmpotter In order to identify a private key for a public key, is it possible to create a unique name for the key pair

**Comment by bmpotter** [comment]
@amaya-Mali Yes, this is a reasonable requirement. We will work on adding this.

**Comment by bmpotter** [comment]
@amaya-Mali This requirement was implemented as part of [issue 78](https://github.com/open-horizon/SDO-support/issues/78)
